ucan-key-support/Cargo.toml

```diff
@@ -25,6 +25,8 @@ anyhow = "1.0.52"
 async-trait = "0.1.52"
 ed25519-zebra = "^3"
 rsa = "0.6"
+# NOTE: upgrade of p256 to >=0.11 blocked by 'did-key'
+p256 = "0.9"
```

Comment:
what are the issues here, maybe worth the upgrade @cdata?

Reply:
Totally open to updating, but it would be nice to get more details!

Reply:
I think the fastest way to debug this is to check out the branch, set the `p256` version to `0.11`, and try building the project. Cargo will give a mostly-helpful output but I find I usually need to do a bunch of digging around on lib.rs and through upstream Cargo.toml files to really trace things down.

Reply:
was now able to update to 0.11